packages/dev/core/src/Buffers/bufferUtils.ts

```diff
@@ -13,7 +254,7 @@ import { Logger } from "../Misc/logger";
  * @param totalVertices number of vertices in the buffer to take into account
  * @param output the output float array
  */
-export function CopyFloatData(
+export function copyFloatData(

```

Comment:
Is it not a breaking change ?

Reply:
Hmm, true, it is. Might've been a nit to make this conform to our naming conventions. @bghgary ?

For what it's worth, I don't see any playgrounds using this, but I'm not sure how reliable of a measure this is to decide if something is being used.
<img width="928" alt="image" src="https://github.com/user-attachments/assets/dbc39d54-3f6d-4691-a042-47b5e651ff6a">

Reply:
Revert for now. We need to decide on a naming convention still.